net/mini_snmpd/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mini_snmpd
+PKG_VERSION:=1.4-rc1
+PKG_RELEASE:=3
+PKG_MAINTAINER:=Luke McKee <hojuruku@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/troglobit/mini-snmpd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=203d92e60ed09466d6676c6ad20ad6cb2ce08a5d
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+# As warned by upstream maintainer and gcc
+PKG_SSP:=0
+PKG_RELRO:=0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mini-snmpd
```

Comment:
Ah damn it you win. Ok RELRO is on, against upstream's wishes.
This is the configure options passed by autofixup.
I'll add another comitt and rebase then we are done. The users on irc think it's cool that mini_snmpd starts by default if installed (now only on the internal network) so I added that as well.
Stand by 1 min for the push.
```
autoreconf: Leaving directory `.'
(cd /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1/./; if [ -x ./configure ]; then find /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1/ -name config.guess | xargs -r chmod u+w; find /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1/ -name config.guess | xargs -r -n1 cp --remove-destination /usr/src/lede/scripts/config.guess; find /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1/ -name config.sub | xargs -r chmod u+w; find /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1/ -name config.sub | xargs -r -n1 cp --remove-destination /usr/src/lede/scripts/config.sub; AR="mips-openwrt-linux-musl-gcc-ar" AS="mips-openwrt-linux-musl-gcc -c -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -iremap /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1:mini_snmpd-1.4-rc1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" LD=mips-openwrt-linux-musl-ld NM="mips-openwrt-linux-musl-gcc-nm" CC="mips-openwrt-linux-musl-gcc" GCC="mips-openwrt-linux-musl-gcc" CXX="mips-openwrt-linux-musl-g++" RANLIB="mips-openwrt-linux-musl-gcc-ranlib" STRIP=mips-openwrt-linux-musl-strip OBJCOPY=mips-openwrt-linux-musl-objcopy OBJDUMP=mips-openwrt-linux-musl-objdump SIZE=mips-openwrt-linux-musl-size CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -iremap /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1:mini_snmpd-1.4-rc1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro " CXXFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -iremap /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1:mini_snmpd-1.4-rc1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro " CPPFLAGS="-I/usr/src/lede/staging_dir/target-mips_34kc_musl-1.1.14/usr/include -I/usr/src/lede/staging_dir/target-mips_34kc_musl-1.1.14/include -I/usr/src/lede/staging_dir/toolchain-mips_34kc_gcc-5.3.0_musl-1.1.14/usr/include -I/usr/src/lede/staging_dir/toolchain-mips_34kc_gcc-5.3.0_musl-1.1.14/include/fortify -I/usr/src/lede/staging_dir/toolchain-mips_34kc_gcc-5.3.0_musl-1.1.14/include " LDFLAGS="-L/usr/src/lede/staging_dir/target-mips_34kc_musl-1.1.14/usr/lib -L/usr/src/lede/staging_dir/target-mips_34kc_musl-1.1.14/lib -L/usr/src/lede/staging_dir/toolchain-mips_34kc_gcc-5.3.0_musl-1.1.14/usr/lib -L/usr/src/lede/staging_dir/toolchain-mips_34kc_gcc-5.3.0_musl-1.1.14/lib -znow -zrelro "  LD="mips-openwrt-linux-musl-gcc" STRIP="/bin/true" CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -iremap /usr/src/lede/build_dir/target-mips_34kc_musl-1.1.14/mini_snmpd-1.4-rc1:mini_snmpd-1.4-rc1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro"  ./configure --target=mips-openwrt-linux --host=mips-openwrt-linux --build=x86_64-pc-linux-gnu --program-prefix="" --program-suffix="" --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --libexecdir=/usr/lib --sysconfdir=/etc --datadir=/usr/share --localstatedir=/var --mandir=/usr/man --infodir=/usr/info --disable-nls    ; fi; )
```